### ISSUE TITLE
`Using fl studio with NVDA

### ISSUE THREAD

**Issue by NVCDevelopmentTeam** [nvaccess/nvda#12484]
I have a friend who is producing electronic music. However, since my friend is blind, he couldn't do it. Because NVDA is not compatible with fl studio by nature. Although, it's very good. I tried contacting that developer but nothing worked. I think, there should be a feature to make NVDA work with fl studio. I find it very useful for electronic music producers and I hope it will be available in future updates.

**Comment by feerrenrut** [comment]
This issue isn't actionable by us. I'm going to close it. It is useful to know there is the desire to use NVDA + FL studio. The initial accessibility work will need to be done within FL studio, until then it is unlikely there is anything we can do.
